includes/REST_API/Lock_Controller.php

```diff
@@ -202,7 +202,12 @@ protected function get_lock( $post_id ) {
 	 * @return true|WP_Error True if the request has read access for the item, WP_Error object otherwise.
 	 */
 	public function get_item_permissions_check( $request ) {
-		return $this->parent_controller->update_item_permissions_check( $request );
+		$check = $this->parent_controller->get_item_permissions_check( $request );
+		if ( is_wp_error( $check ) ) {
+			return $check;
+		}
+
+		return $this->get_post_type_cap( $this->post_type, 'edit_posts' );
```

Comment:
Fallback to see if the user can access the post type in general.

Reply:
This should check for the specific capability for the supplied post ID (`$request['id']`)

Reply:
No it shouldn't. That is what was happen in `$this->parent_controller->update_item_permissions_check`. [See](https://github.com/WordPress/wordpress-develop/blob/31a6cd2f781ebe12f2858fd389cd95dccb23cbd5/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L1632-L1640). 

To see a lock, you only need to be able to edit an story, not this story.